indico/modules/events/layout/__init__.py

```diff
@@ -104,3 +104,9 @@ class ImagesFeature(EventFeature):
     @classmethod
     def is_default_for_event(cls, event):
         return event.getType() == 'conference'
+
+
+@signals.event_management.clone.connect
+def _get_layout_and_menu_cloner(event, **kwargs):
+    from indico.modules.events.layout.clone import MenuAndLayoutCloner
+    return MenuAndLayoutCloner(event)
```

Comment:
Really, MenuAndLayout? Why not just LayoutCloner? Menu is part of the layout module